src/components/createPatientModal/createPatientModal.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import Button from 'react-bootstrap/esm/Button'
+import Form from 'react-bootstrap/esm/Form'
+import Modal from 'react-bootstrap/esm/Modal'
+
+function CreatePatientModal(props) {
+  console.log(props)
```

Comment:
Do we still need this console log? It's easy to forget about them once they are in master.